tests/src/Kernel/fixtures/markup_rendering_patterns/listing.yml

```
@@ -82,6 +82,7 @@ listing_default_1_col:
       'p.card-text.mb-3': 4
       'span.me-4': 4
       'span.mb-2': 8
+      'h5.card-title': 4
```

Add link attributes testing: 
```suggestion
      'h5.card-title': 4
      'a.d-inline-block': 1
```